force-app/main/default/lwc/serviceScheduleCreator/__tests__/serviceScheduleCreator.test.js

```diff
@@ -21,13 +21,22 @@ describe("c-service-schedule-creator", () => {
         });
     });
 
-    it("modal is not displayed and element is accessible", () => {
+    it("modal is displayed and element is accessible", () => {
+        const HEADER_STRING = "New Service Schedule";
         document.body.appendChild(element);
 
         return global.flushPromises().then(async () => {
             const modal = element.shadowRoot.querySelector("c-modal");
-            // Modal will only display after data is loaded
-            expect(modal).toBe(null);
+
+            const h2Container = modal.shadowRoot.querySelector("h2");
+            let slot = document.createElement("span");
+            slot.textContent = HEADER_STRING;
```

Comment:
Great idea but I think we need to think about this for a bit. We are actually inserting a header here but maybe what we want to test is that when step1 loads it is accessible. I will make a note that we should iterate on this test.